test/integration/test_raw_api.py

```diff
@@ -281,108 +283,108 @@ def test_file_upload(file_dict):
     pass
 
 
-def test_list_file_versions(raw_api, api_url, account_auth_token, bucket_id, file_dict, file_name):
+def test_list_file_versions(raw_api, api_url, account_auth_token, bucket_id, file_dict):
     list_versions_dict = raw_api.list_file_versions(api_url, account_auth_token, bucket_id)
-    assert [file_name] == [f_dict['fileName'] for f_dict in list_versions_dict['files']]
+    assert [TEST_FILE_NAME] == [f_dict['fileName'] for f_dict in list_versions_dict['files']]
 
 
 def test_download_file_by_id_with_auth(
-    raw_api, download_url, file_dict, account_auth_token, file_contents
+    raw_api, download_url, file_dict, account_auth_token,
 ):
     url = raw_api.get_download_url_by_id(download_url, file_dict['fileId'])
     with raw_api.download_file_from_url(account_auth_token, url) as response:
-        data = next(response.iter_content(chunk_size=len(file_contents)))
-        assert data == file_contents, data
+        data = next(response.iter_content(chunk_size=len(TEST_FILE_CONTENTS)))
+        assert data == TEST_FILE_CONTENTS, data
 
 
-def test_download_file_by_id_no_auth(raw_api, download_url, file_dict, file_contents):
+def test_download_file_by_id_no_auth(raw_api, download_url, file_dict):
```

Comment:
file_dict is an unused variable here and in a few more tests. What I think you want to do is a class-based test which inherits from a `FileHandlingHelper` which uploads a file and assigns it to a `self.file_dict` in a `setUp` method

Reply:
Why so complicated? I need a file uploaded for the test. I require it in test inputs.